toolsets/oracle-operator/primitive.go

```diff
@@ -27,7 +27,8 @@ func queryPrimitive(
 ) {
 	logger := ctx.Logger()
 	logger.Debug("query primitive")
-	logger.Debug(payload.Contract, payload.Function)
+	logger.Debug("contract", payload.Contract)
```

Comment:
The first argument will be the log entry body and all other arguments become kv-pairs.

```go
logger.Debug("body", "k1", "v1", "k2", "v2")  // => [DEBUG]body k1=v1 k2=v2
```